controllers/ControllerUser.class.php

```diff
@@ -49,8 +49,29 @@ public function logout() {
 
     public function settings() {
         $error = null;
+        if (!isset($_SESSION['userId'])) {
+            throw new Exception('You need to be connected to access the settings !');
+        }
+        $this->_userManager = new UserManager();
+        $user = $this->_userManager->getByUserId($_SESSION['userId']);
+        if (isset($_POST['editProfil'])) {
```

Comment:
```suggestion
        if (isset($_POST['editProfil'])) {
```
Profil is a french word, use Profile.